README.md

```diff
@@ -76,23 +67,23 @@ $ minishift start --memory=8GB
 ```
 
 From this command output you need:
- - Minishift master url. In this case it's `https://192.168.42.159:8443`. Let's call it 'CHE_INFRA_KUBERNETES_MASTER__URL'. We can store this variable in the terminal session to use it for next commands:
+ - Minishift master url. In this case it's `https://192.168.42.159:8443`. Let's call it 'Che_INFRA_KUBERNETES_MASTER__URL'. We can store this variable in the terminal session to use it for next commands:
```

Comment:
Che_INFRA_KUBERNETES_MASTER__URL - is this variable intentionally have `Che_` instead of `CHE_`